offchain-modules/cli/src/commands/mod.rs

```diff
@@ -29,19 +29,16 @@ pub async fn handler(opt: Opts) -> Result<()> {
         // verify ckb spv proof && unlock erc20 token.
         SubCommand::Unlock(args) => unlock_handler(args),
         SubCommand::TransferFromCkb(args) => transfer_from_ckb_handler(args),
+
+        SubCommand::EthRelay(args) => eth_relay_handler(args),
+        SubCommand::CkbRelay(args) => ckb_relay_handler(args).await,
     }
 }
 
 pub async fn approve_handler(args: ApproveArgs) -> Result<()> {
     debug!("approve_handler args: {:?}", &args);
-    if args.from.len() != ETH_ADDRESS_LENGTH {
-        return Err(Error::msg("invalid from address"));
-    }
-    if args.to.len() != ETH_ADDRESS_LENGTH {
-        return Err(Error::msg("invalid to address"));
-    }
-    let from: H160 = H160::from_slice(hex::decode(args.from)?.as_slice());
-    let to = H160::from_slice(hex::decode(args.to)?.as_slice());
+    let from = convert_eth_address(args.from.as_str())?;
```

Comment:
```suggestion
    let from = convert_eth_address(&args.from)?;
```

This way is simpler. The way now works though.